README.md

```diff
@@ -1,53 +1,89 @@
-# ELISA Automotive WG - Navigation and file locations
-## Mailing list, Calendar etc
+# ELISA Automotive WG
+## Mission statement
+The automotive workgroup discusses the conditions and prerequisites the automotive sector needs to integrate Linux into a safety critical system.
+ 
+We focus on actual use cases from the Automotive domain to derive the technical requirements to the kernel as a basis for investigation within the Architecture Workgroup and to serve as a blueprint for actual projects in the future.

```

Comment:
Change "Architecture Workgroup" to "Automotive Workgroup"?

Reply:
Safety Architecture Workgroup will make it clearer.
Other than this Architecture will remain, as it illustrates the interaction.

Reply:
@Jochen-Kall  are you willing to update just with the Safety in front of Architecture Workgroup?

Reply:
Sure, and done :)